main-client.js

```diff
@@ -1,13 +1,15 @@
 import './check-npm.js';
 
-import { createNetworkInterface } from 'apollo-client';
+import { createNetworkInterface, createBatchingNetworkInterface } from 'apollo-client';
 import { Accounts } from 'meteor/accounts-base';
 import { _ } from 'meteor/underscore';
 
 const defaultNetworkInterfaceConfig = {
-  path: '/graphql',
-  options: {},
-  useMeteorAccounts: true
+  path: '/graphql', // default graphql server endpoint
+  opts: {}, // additional fetch options like `credentials` or `headers`
+  useMeteorAccounts: true, // if true, send an eventual Meteor login token to identify the current user with every request
+  batchingInterface: false, // if true, use a BatchedNetworkInterface instead of a NetworkInterface
```

Comment:
maybe we should use batching by default? seems like it would be beneficial to most